test-network-function/suite_test.go

```diff
@@ -41,8 +41,11 @@ import (
 )
 
 const (
+	claimFileName                 = "claim.json"
```

Comment:
Nothing to do for this PR, but in the future we should probably generate this instead with some sort of timestamp to prevent overwriting past attempts.